etcd/configure_coreos_components_to_use_tls_etcd.md

```diff
@@ -0,0 +1,114 @@
+# Configure CoreOS Components to Use TLS etcd
+
+If you already have finished [HTTP to HTTPS migration][etcd-live-http-https] and would like to switch all your CoreOS components to use secure HTTPS connection with etcd cluster please follow instruction below.
```

Comment:
This document explains how to configure all CoreOS components to use secure HTTPS connections to an etcd cluster. The target etcd cluster must already be using HTTPS for its own communications, as explained in the [etcd HTTP to HTTPS migration guide](etcd-live-http-https.md).